src/Pages/AboutUsManager/AboutUsManagerModal.js

```diff
@@ -0,0 +1,312 @@
+    await props.handleDelete();
+    window.location.reload();
+    toggleConfirmationModal();
+    props.toggle();
+  }
+
+  const confirmModalProps = {
+    headerText: `Delete ${props.name}?`,
+    bodyText: 'This officer will be permanently removed from the system.',
+    toggle: toggleConfirmationModal,
+    handleConfirmation: handleDeletion,
+    open: confirmationModal
+  };
+  const inputMatrixExec = [
+    [
+      {
+        addon: 'Name*',
+        type: 'text',
+        defaultValue: props.name,
+        placeholder: 'e.g. John Doe',
+        maxlength: '24',
+        handleChange: e => setName(e.target.value)
+      },
+      {
+        addon: 'Email*',
+        type: 'email',
+        defaultValue: props.email,
+        handleChange: e => setEmail(e.target.value)
+      }
+    ],
+    [
+      {
+        addon: 'LinkedIn Profile*',
+        type: 'url',
+        placeholder:'Enter URL Here',
+        defaultValue: props.linkedin,
+        handleChange: e => setLinkedin(e.target.value)
+      },
+      {
+        addon: 'Position*',
+        type: 'text',
+        defaultValue: props.position,
+        maxlength: '20',
+        handleChange: e => setPosition(e.target.value)
+      }
+    ],
+    [
+      {
+        addon: 'Description* (180 characters max)',
+        type: 'textarea',
+        rows: 5,
+        placeholder: 'Enter Description',
+        defaultValue: props.quote,
+        onChange: e => setQuote(e.target.value),
+        maxlength: '180'
+      }
+    ],
+    [
+      {
+        addon: 'Profile Picture*',
+        type: 'url',
+        placeholder:'Enter URL Here',
+        defaultValue: props.pictureUrl,
+        handleChange: e => setPictureUrl(e.target.value)
+      }
+    ]
+  ];
+
+  const inputMatrixOfficers = [
+    [
+      {
+        addon: 'First Name*',
+        type: 'text',
+        defaultValue: props.name,
+        placeholder: 'e.g. John D.',
+        maxlength: '24',
+        handleChange: e => setName(e.target.value)
+      },
+      {
+        addon: 'Email*',
+        type: 'email',
+        defaultValue: props.email,
+        handleChange: e => setEmail(e.target.value)
+      }
+    ],
+    [
+      {
+        addon: 'LinkedIn Profile',
+        type: 'url',
+        placeholder:'Enter URL Here',
+        defaultValue: props.linkedin,
+        handleChange: e => setLinkedin(e.target.value)
+      }
+    ],
+    [
+      {
+        addon: 'Profile Picture*',
+        type: 'url',
+        placeholder: 'Enter URL Here',
+        defaultValue: props.pictureUrl,
+        handleChange: e => setPictureUrl(e.target.value)
+      }
+    ]
+  ];
+
+  async function handleSubmission() {
+    const officerFields = {
+      name,
+      email,
+      linkedin,
+      team,
+      position,
+      quote,
+      pictureUrl
+    };
+    await props.handleSubmit({...officerFields});
+    window.location.reload();
+    props.toggle();
+  }
+
+  function requiredFieldsFilledIn() {
```

Comment:
there is a lot of duplication in this function. furthermore empty strings and undefined variables are interpreted as false.

I would suggest initially checking fields that we know must be filled in like:

```js
function requiredFieldsFilledIn() {
  const nameEmailTeamPicture = name && email && team && pictureUrl;
  const linkedInQuotePosition = linkedin && quote && position;
}
```

then in your `if` statements you can do:

```js
// case 1 and 2
if (team === 'executive' && props.modalState === officerModalState.EDIT
    || props.modalState === officerModalState.SUBMIT) {
  return nameEmailTeamPicture && linkedinQuotePosition;
} else if (props.modalState === officerModalState.SUBMIT
          || props.modalState === officerModalState.EDIT) { // case 3 and 4
  return nameEmailTeamPicture;
}
```